meetings/working-groups/roles/extensions-2024-01-25.md

```diff
@@ -0,0 +1,72 @@
+# Extensions 2024-01-25
+
+### Scope and priorities for C#13
+static without inheritance -> instance -> inheritance support -> (out of scope: interfaces)
+
+### Tracking open issues and status with LDM
+We're using TODO/TODO2/TODO3 tags in the speclet to track and prioritize open issues.  
+I'll add "TODO review with LDM" tags to mark sections that need reviewing. Those will be removed when reviewed.  
+We'll use a PR format to bring speclet updates to review to LDM.  
+
+### Adjusting lookup for members of underlying type
+The current rules state that we look in the underlying type only when the lookup failed in the extension type.  
+Those rules are inadequate for method group, invocation and indexer access scenarios as illustrated by example below.  
+We instead want to treat the underlying type as a base type. It still comes into play last (meaning after all base extensions) but not conditionally.  
+This allows for a method group with both methods from extension and underlying type.  
+
+```
+var a1 = EDerived.M2; // method group should incorporate members of EDerived, EBase and C
```

Comment:
`EBase.M2` is `private`
`EDerived` has a `Method` (also `private`) but not `M2`

Based on the comment I expected to see `M2` from a few types but only see one that would be applicable here. Not sure how `Method` factors into this.